contracts/core/Dispatcher.sol

```diff
@@ -453,25 +481,33 @@ contract Dispatcher is IbcDispatcher, IbcEventsEmitter, Ownable, Ibc {
         emit SendPacket(sender, channelId, packet, sequence, timeoutTimestamp);
     }
 
-    function _verifyConnectIbcChannelProof(
+    function _connectChannel(
+        IbcChannelReceiver portAddress,
         CounterParty calldata local,
         string[] calldata connectionHops,
         ChannelOrder ordering,
-        bool isChanConfirm,
-        CounterParty calldata counterparty,
-        Ics23Proof calldata proof
+        bool feeEnabled,
+        CounterParty calldata counterparty
     ) internal {
-        consensusStateManager.verifyMembership(
-            proof,
-            channelProofKey(local.portId, local.channelId),
-            channelProofValue(
-                isChanConfirm ? ChannelState.CONFIRM_PENDING : ChannelState.ACK_PENDING,
-                ordering,
-                local.version,
-                connectionHops,
-                counterparty
-            )
+        portAddress.onChannelConnect(local.channelId, counterparty.version);
```

Comment:
These callbacks have slightly different naming than the one in the [Polymer Docs here](https://docs.polymerlabs.org/docs/learn/concepts/ibc/#channels-ics-4) - not sure if this is something we should change in this PR (considering that there are already other interface changes) or in another PR or if this is something we want to change at all

Reply:
I asked that question myself and thought it would be easier to have the one `onChannelConnect` instead of splitting it in two. But given the docs you pointed out, I think you have a valid point. Let me split them too and be consistent throughout

Reply:
awesome, thank you for implementing this!